docs/node-operators/get-started/setup-validator.md

```diff
@@ -24,6 +24,85 @@ To update your node configuration, such as to set up ports for the APIs, edit th
 YOUR_VEGA_HOME_PATH/config/node/config.toml
 ```
 
+## Initialise Visor for protocol upgrades
```

Comment:
```suggestion
## Initialise Visor (for smooth protocol upgrades)
```

maybe?

Reply:
Maybe without smooth. But the protocol upgrade this way sounds a bit confusing. Or maybe it's just me.